src/components/RangeSlider/components/DualThumb/DualThumb.scss

```diff
@@ -2,18 +2,27 @@
 @import '../../RangeSlider';
 
 $stacking-order: (
-  output: 10,
+  output: 20,
   input: 20,
```

Comment:
This could be achieved by inverting these values instead. If you want the output to stack over the input, it only needs a larger value. Make this:

```
input: 10,
output: 20,
```